build.sbt

```diff
@@ -0,0 +1,29 @@
+val tapirVersion = "1.1.0"
+
+lazy val rootProject = (project in file("."))
+  .enablePlugins(ScalaxbPlugin)
+  .settings(
+    Seq(
+      name := "tapir-scalaxb-example",
+      version := "0.1.0-SNAPSHOT",
+      organization := "com.softwaremill.tapir.scalaxb.example",
+      scalaVersion := "2.13.8",
+      libraryDependencies ++= Seq(
+        "com.softwaremill.sttp.tapir" %% "tapir-http4s-server" % tapirVersion,
+        "org.http4s" %% "http4s-blaze-server" % "0.23.12",
+        "com.softwaremill.sttp.tapir" %% "tapir-swagger-ui-bundle" % tapirVersion,
+        "com.softwaremill.sttp.tapir" %% "tapir-json-circe" % "1.1.0",
+        "ch.qos.logback" % "logback-classic" % "1.4.1",
```

Comment:
and this one? :)

Reply:
Yes, without it, after starting the project, we would see a "failed to load logger" message.